calib:
  cam1_cam2:
    matching:
      match_th_dist: 500.0
      normal_radius: 50.0
      extract_objets: false
      kp_corr_rej: true
    transform_estimation:
      use_points: false
      use_planes: true
      compute_rotation: true
      compute_translation: true
      project_points: false
    merging:
      enable_cutting: true
      y_pose: 40.0
      width: 500.0
    tf_filtering:
      cam1_cam2:
        rejection: false
        average: true
        filter_weight: 3.0
    scene_icp:
      tf_filtering:
        rejection: false
        average: true
        filter_weight: 3.0
  scene:
    icp:
      enable: true
      subsize: 40
      autom: false
      max_dist: 500
      max_it: 1000
      epsilon: 6